smithy-typescript-ssdk-libs/server-apigateway/src/lambda.ts

```diff
@@ -33,6 +58,7 @@ export function convertEvent(event: APIGatewayProxyEventV2): HttpRequest {
   });
 }
 
+export const convertVersion2Response = convertResponse;
 export function convertResponse(response: HttpResponse): APIGatewayProxyResultV2 {
```

Comment:
are we going to remove `convertResponse` after migration demo package?

Reply:
Yeah, guess we can since this has no users.